CumulusCI `error` Command (#1502)
SFDO-Tooling/CumulusCI: docs/features.rst

* See `this link <https://ntotten.com/2018/05/11/convert-metadata-to-source-format-while-maintain-git-history/>`_ for some tips on preserving git history while converting your source format.

Working with Errors
===================

> Logfiles

Reviewer: ```suggestion
Log files
```
I think this is [two words](https://en.wikipedia.org/wiki/Log_file)

--------

> CumulusCI creates a logfile every time a cci command is run. The only exception to this is when the cci gist command is run. Logfiles are stored under `~/.cumulusci/logs`.

Reviewer: ```suggestion
CumulusCI creates a log file every time a cci command besides ``gist`` is run. Log files are stored under ``~/.cumulusci/logs``.
```

RST needs double backticks for inline code. 😒

Viewing Stacktraces
-------------------

> If you encounter an error and want more information on what went wrong, you can use `cci error info` to display the last 30 lines of the stacktrace (if present) from the last command you executed in CumulusCI.

Reviewer: ```suggestion
If you encounter an error and want more information on what went wrong, you can use ``cci error info`` to display the last *n* lines (30, by default) of the stacktrace (if present) from the last command you executed in CumulusCI.
```

> You can include the option num_lines argument if you want to customize how much of the stacktrace you see.

Reviewer: ```suggestion
You can include the option ``max_lines`` argument if you want to customize how much of the stacktrace you see.
```

> You can include the option num_lines argument if you want to customize how much of the stacktrae you see.

Reviewer: ```suggestion
You can include the option num_lines argument if you want to customize how much of the stacktrace you see.
```
